synapse/handlers/deactivate_account.py

```diff
@@ -120,6 +120,10 @@ def deactivate_account(self, user_id, erase_data, id_server=None):
         # parts users from rooms (if it isn't already running)
         self._start_user_parting()
 
+        # Reject all pending invites for the user, so that it doesn't show up in the
+        # invitees list of rooms.
```

Comment:
```suggestion
        # invitee's list of rooms.
```

Reply:
Actually this whole comment is wrong, I'll rewrite it.